CHANGELOG.md

```diff
@@ -9,7 +9,7 @@
 * None.
 
 ### Breaking changes
-* None.
+* The websocket error codes `websocket_client_too_old`, `websocket_client_too_new`, and `websocket_protocol_mismatch` along with their C API constants were removed. These corresponded to errors the legacy C++ server could have sent, but the baas sync server never did. Any platform networking implementations that surfaced these errors can report a `websocket_fatal_error` instead if an unknown error occurs during the websocket handshake. If a client connects that is too old or too new, it will finish the websocket handshake and then receive an in-band sync `ERROR` message that will be handled by the sync error handler. [PR #7917](https://github.com/realm/realm-core/pull/7917)
```

Comment:
I guess it's technically a breaking change, but I doubt any of the SDKs with platform networking care about this error (and I doubt they've ever seen it).